src/scripts/browser.js

```diff
@@ -292,7 +292,7 @@ define([], function () {
     }
 
     if (typeof document !== 'undefined') {
-        if (('ontouchstart' in window) || window.DocumentTouch && document instanceof DocumentTouch) {
+        if (('ontouchstart' in window) || (navigator.MaxTouchPoints > 0)) {
```

Comment:
I think it should be `navigator.maxTouchPoints`, but didn't check.
https://developer.mozilla.org/en-US/docs/Web/API/Navigator/maxTouchPoints